pkg/js/transformer/dsltojson.ts

```diff
@@ -116,6 +115,15 @@ class OpenFgaDslListener extends OpenFGAListener {
       }
     }
     if (relationDef) {
+      const relationName = ctx.relationName().getText();
```

Comment:
Why not do:

```js
if (!ctx.relationName()) {
  return;
}
```

At the start?

There is no case where you have no relation name or no relation definition and where you should proceed as usual